src/login/pages/Login.tsx

```diff
@@ -28,98 +31,115 @@ export default function Login(props: PageProps<Extract<KcContext, { pageId: "log
-                            <div className="links">
-                                <p>
-                                    <a href={url.loginResetCredentialsUrl}>
-                                        Forgot password?
-                                    </a>
+                                    {messagesPerField.existsError("username", "password") && (
+                                        <span id="input-error" className={kcClsx("kcInputErrorMessageClass")} aria-live="polite">
+                                            {messagesPerField.getFirstError("username", "password")}
+                                        </span>
+                                    )}
+                                </>
+                                <div className={kcClsx("kcFormGroupClass")}>
+                                    <label style={{ marginTop: '20px', fontSize: '0.8rem', color: '#1E678F' }} htmlFor="password" className={kcClsx("kcLabelClass")}>
+                                        Password
+                                    </label>
+                                    <PasswordWrapper kcClsx={kcClsx} i18n={i18n} passwordInputId="password">
+                                        <input
+                                            tabIndex={3}
+                                            id="password"
+                                            className={kcClsx("kcInputClass")}
+                                            name="password"
+                                            type="password"
+                                            autoComplete="current-password"
+                                            aria-invalid={messagesPerField.existsError("username", "password")}
+                                        />
+                                    </PasswordWrapper>
+                                    {usernameHidden && messagesPerField.existsError("username", "password") && (
+                                        <span id="input-error" className={kcClsx("kcInputErrorMessageClass")} aria-live="polite">
+                                            {messagesPerField.getFirstError("username", "password")}
+                                        </span>
+                                    )}
+                                </div>
+                                <div className="links">
+                                    <p>
+                                        <a href={url.loginResetCredentialsUrl}>
+                                            Forgot password?
+                                        </a>
 
-                                    <br />
+                                        <br />
 
-                                    <a href={url.loginResetCredentialsUrl}>
-                                        Staff login
-                                    </a>
-                                </p>
+                                        <a href={url.loginResetCredentialsUrl}>
```

Comment:
Was taking a quick look at some of the url properties.  Looks like there's one called `ssoLoginInOtherTabsUrl` have you tried using that?